mantle/kola/tests/ignition/empty.go

```diff
@@ -45,5 +45,9 @@ func init() {
 	})
 }
 
-func empty(_ cluster.TestCluster) {
+func empty(c cluster.TestCluster) {
+	m := c.Machines()[0]
+	// check that the test harness correctly skipped passing SSH keys
+	// via Ignition
+	c.MustSSH(m, "[ ! -e ~/.ssh/authorized_keys.d/ignition ]")
```

Comment:
The `coreos.ignition.ssh.key` also uses this function and that function is now failing.

- https://jenkins-fedora-coreos.apps.ocp.ci.centos.org/blue/organizations/jenkins/kola-aws/detail/kola-aws/462/pipeline

```
--- FAIL: coreos.ignition.ssh.key (426.60s)

        cluster.go:164: "[ ! -e ~/.ssh/authorized_keys.d/ignition ]" failed: output , status Process exited with status 1

```

Reply:
https://github.com/coreos/coreos-assembler/pull/2258